lib/install/config/shared.js

```diff
@@ -5,13 +5,35 @@ const path = require('path')
 const process = require('process')
 const glob = require('glob')
 const extname = require('path-complete-extname')
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+const ManifestPlugin = require('webpack-manifest-plugin')
 
 let distDir = process.env.WEBPACK_DIST_DIR
+const devHost = process.env.DEV_HOST
+const devPort = process.env.DEV_PORT
 
 if (distDir === undefined) {
   distDir = 'packs'
 }
 
+let publicPath = `/${distDir}/`
+const fileLoaderExtensions = /\.(jpe?g|png|gif|svg|eot|svg|ttf|woff|woff2)$/i
+const fileLoaderConfig = {
+  loader: 'file-loader',
+  options: {
+    publicPath,
+    name: '[name].[ext]'
+  }
+}
+
+if (devHost && devPort) {
```

Comment:
All this logic should go into a file that is called like `webpack.development.hot.config`.

I like to have a file that does simple creation of the assets statically for development mode. Hitting CMD-R has been better for us on big projects.